mqttListener:
  anchorConfig:
    msgType: anchorConfig
    listenTo: "mqtt_anchorConfig_output"
    pubTo: "anchorConfig"